weave/trace_server/clickhouse_schema.py

```diff
@@ -150,4 +150,5 @@ class SelectableCHObjSchema(BaseModel):
     base_object_class: typing.Optional[str]
     digest: str
     version_index: int
+    version_count: int
```

Comment:
we already calculate this, and it would be great from a performance standpoint. Is it too confusing for users though? on the object schema?